xbmc/interfaces/legacy/Control.cpp

```diff
@@ -537,6 +537,7 @@ namespace XBMCAddon
     ControlRadioButton::ControlRadioButton(long x, long y, long width, long height, const String& label,
                                            const char* focusOnTexture,  const char* noFocusOnTexture,
                                            const char* focusOffTexture, const char* noFocusOffTexture,
+                                           const char* focusTexture, const char* noFocusTexture,
```

Comment:
Since I suppose this is the opposite to focusoff, can it be named focuson? Or is this naming for compatibility reasons?

Reply:
'on' and 'off' refer to the state of the radiobutton.
focus and nofocus refer to wether the button is focused or not.
so these are different things...